.golangci.yaml

```diff
@@ -1,3 +1,6 @@
+run:
+  modules-download-mode: readonly
```

Comment:
Why is it necessary?

Reply:
It may not be necessary anymore. I was getting linter errors in CI, but not locally, and after googling possible cause I found that it may be trying to update vendored modules.

The issue may also have been caused by using golang 1.19 in CI, when the code needs 1.20.
I will try to remove it.